Examples/Rosbridge-Publish-Float/Publish_float/Publish_float.ino

```diff
@@ -0,0 +1,94 @@
+#include <WiFi.h>
+#include <WebSocketClient.h> 
+
+// Set these to your desired credentials.
+const char *ssid = "";
+const char *password = "";
+
+// Update with ROS master IP and rosbridge port (default 9090)
+const char *host_ip = "172.31.1.21";
+const int host_port = 9090;
+char host[] = "ws://172.31.1.21:9090";
+
+char path[] = "/";
+
+WiFiClient client;
+WebSocketClient webSocketClient;
+
+bool testBit = true;
+bool toggleBit = false;
+
+
+void setup() {
+  
+  Serial.begin(9600);
+  delay(1000);
+  Serial.flush();
+  Serial.println("Setup..");
+  delay(10);
+  WiFi.begin(ssid, password);
+   
+  while (WiFi.status() != WL_CONNECTED) {
+      delay(500);
+      Serial.print(".");
+  }
+   
+  Serial.println("");
+  Serial.println("WiFi connected");
+  Serial.println("IP address: ");
+  Serial.println(WiFi.localIP());
+   
+  delay(5000);
+
+  if(client.connect(host_ip, host_port)) {
+    Serial.println("Connected");
+  }else {
+      Serial.println("Connection failed.");
+      while(1) {
+      // Hang on failure
+    }
+  }
+
+  webSocketClient.path = path;
+  webSocketClient.host = host;
+  if (webSocketClient.handshake(client)) {
+    Serial.println("Handshake successful");
+  } else {
+    Serial.println("Handshake failed.");
+    while(1) {
+      // Hang on failure
+    } 
+  }
+  
+}
+
+void loop() {
+  // put your main code here, to run repeatedly:
+  Serial.println("Loopin..");
+  
+  toggleBit = !toggleBit;
+  
+  if (client.connected()) {
+    if (testBit){
+      String msg = "{\"op\": \"advertise\", \"topic\": \"/panda_touching\", \"type\": \"std_msgs/float32\"}";
+      Serial.println(msg);
+      webSocketClient.sendData(msg);
+      testBit=false;
+    } else {
+      String msg = "{\"op\": \"publish\", \"topic\": \"/panda_touching\", \"msg\": {\"data\":";
+      if (toggleBit){
```

Comment:
looks all good, to make it ready for the Panda_FSR only the toggle bit has to be removed i guess

Reply:
that were not exactly the lines I was referencing to but ok :D